lib/modules/manager/pip-compile/extract.spec.ts

```diff
@@ -246,26 +246,33 @@ describe('modules/manager/pip-compile/extract', () => {
   });
 
   it('return sorted package files', async () => {
-    fs.readLocalFile.mockResolvedValueOnce(
-      getSimpleRequirementsFile('pip-compile --output-file=4.txt 3.in', [
-        'foo==1.0.1',
-      ]),
-    );
-    fs.readLocalFile.mockResolvedValueOnce('-r 2.txt\nfoo');
-    fs.readLocalFile.mockResolvedValueOnce(
-      getSimpleRequirementsFile('pip-compile --output-file=2.txt 1.in', [
-        'foo==1.0.1',
-      ]),
-    );
-    fs.readLocalFile.mockResolvedValueOnce('foo');
+    // eslint-disable-next-line require-await,@typescript-eslint/require-await
```

Comment:
I just think this is the cleanest way to do it.
- `Promise.resolve(...)` is a bit verbose and less clear, to me as a less-experienced TS dev at least, what it's doing.  It's just one more thing to think about when trying to understand the code.
- `await Promise.resolve()` is just disabling the warning in a less clear way.  It would need an extra comment explaining why its there, negating any possible brevity benefit.
- `eslint-disable-next-line` is clear in what it's doing and clear that it's something I did on purpose.

I can certainly change it if you would like, but this seemed to me to be the best option.

Reply:
@viceice Just a quick bump, since it's been a couple of days.  Any thoughts on this?

Reply:
🤔 can we simply set  return type to any? would this work without eslint annotation? if not leave as is

Reply:
👍 The type-checker seems happy with that